standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1014,6 +1014,7 @@ struct OpenTxnsResponse {
 struct AbortTxnRequest {
     1: required i64 txnid,
     2: optional string replPolicy,
+    3: optional TxnType txn_type = TxnType.DEFAULT,
```

Comment:
we seem to use camelcase in this file - can we continue to do that?

I'm just curious when a field is marked as optional - and also has a default value ; will it still be optional?

Reply:
Same syntax we use for openTxnRequest as well. So, I thought we should make it same of this as well.